go.mod

```diff
@@ -111,3 +112,5 @@ replace (
 
 	kubevirt.io/client-go => ./staging/src/kubevirt.io/client-go
 )
+
+go 1.13
```

Comment:
this part of go.mod needs to be removed. it slips in occasionally

Reply:
i fixed this myself on this pr since it was super simple.

Reply:
Thanks, IDE...